### PR TITLE
Create QualityPlan.md

### PR DESCRIPTION
**Pull request by Chris-Hicks**
Initial attempt at a new Quality Plan as we transition away from Technical Engineering Services

### CODE REVIEW

File: QualityAssurance/QualityPlan.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Additional Responsibilities
+
+### Quality Plan Ownership
+
+The Manufacturing Applications Manager is responsible for the delivery of this Quality Plan.  All Manufacturing Applications personnel are responsible for its implementation through the utilisation of Quality Management System.
+
+### Communications
+
+Communications and/or review meetings will be held with user representatives in each plant area to discuss work undertaken and development roadmaps.  The following information will be provided to aid these discussions:
```
Comment (steve-wells) [inline QualityAssurance/QualityPlan.md]: Should the frequency of review meetings be defined here?

File: QualityAssurance/QualityPlan.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+# Quality Plan Coverage
+
+## Activities Covered
+
+This Quality Plan covers the following activities:
+
+- Support – Work that pertains to the continual running of Manufacturing Applications systems; examples include: bug fixes, data repair and information request.
+- Features – Work that changes the operation of a Manufacturing Applications system in such a way it will exceed or no longer meet original design; examples include: new functionality, removal of functionality and additional displays.
+- Management of External Companies
```
Comment (Mike-Farnhill) [inline QualityAssurance/QualityPlan.md]: Any old external company... Not sure how would word if going to clarify, must it be Manufacturing Applications instigated external company, or user instigated external companies included?

File: QualityAssurance/QualityPlan.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- Out of hours support – This is subject to the availability of personnel and is not a standby arrangement.  In order to request out of hours support, a callout should be placed through Gas Control on 2073 requesting assistance for the relevant area.
+
+These activities can be undertaken in all areas of the British Steel business so long as they are deemed to align with the remit of the department.
+
+## Service Provided
+
+The Manufacturing Applications department will maintain, run, develop and backup all systems under its ownership.  These systems are owned by the Manufacturing Applications department.  Personnel outside of the Manufacturing Applications team are users of these systems.
```
Comment (Mike-Farnhill) [inline QualityAssurance/QualityPlan.md]: Will maintain - does this include hardware? Also where third party agreed to only backup and/or run but not develop, implies shall be developed?

File: QualityAssurance/QualityPlan.md (hunk 1 above)
Comment (Chris-Hicks) [inline QualityAssurance/QualityPlan.md]: Agreed but the frequency varies across plant areas and is determined by their level of involvement.  Will highlight frequency is determined by them.

File: QualityAssurance/QualityPlan.md (hunk 2 above)
Comment (Chris-Hicks) [inline QualityAssurance/QualityPlan.md]: Good point.  I'll add along the lines of instigated by ourselves or by agreement with the Manufacturing Applications Manager

File: QualityAssurance/QualityPlan.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# **Manufacturing Applications Quality Plan**
```
Comment (Mike-Farnhill) [inline QualityAssurance/QualityPlan.md]: Always mentioned as systems - Manufacturing Applications will be hardware support also? Or does clarification that it is the applications being primarily supported?

File: QualityAssurance/QualityPlan.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Communications and/or review meetings will be held with user representatives in each plant area to discuss work undertaken and development roadmaps.  The frequency of these review meetings will be determined by the user representatives and the following information will be provided to aid these discussions:
+
+- Closed issues/requests
+- Raised issues/requests
+- Current development roadmap and anticipated release dates
+
+### Competence &amp; Training
+
+Manufacturing Applications personnel are to undertake relevant safety inductions and refreshers that are identified in customer procedures.  Role based training will be identified by the Manufacturing Applications Manager.  The provision, rollout and tracking of this training will be managed by the Manufacturing Applications team.
```
Comment (Mike-Farnhill) [inline QualityAssurance/QualityPlan.md]: User has become a customer here!

File: QualityAssurance/QualityPlan.md (hunk 3 above)
Comment (Chris-Hicks) [inline QualityAssurance/QualityPlan.md]: I should maybe clarify that we are solely applications.  

In terms of the development comment I'm going to leave as is.  I highlight that all developments are at our direction and discretion.  This includes the option of no development.

File: QualityAssurance/QualityPlan.md (hunk 5 above)
Comment (Chris-Hicks) [inline QualityAssurance/QualityPlan.md]: Good spot

File: QualityAssurance/QualityPlan.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+### Compliments &amp; Complaints
+
+Users have the opportunity to raise an official compliment or complaint.  This will need to be submitted in writing to the Manufacturing Applications Manager who will ensure it is recorded within the Quality Management System.  These will be communicated and reviewed in the monthly team meeting with corrective actions raised appropriately.
```
Comment (Mike-Farnhill) [inline QualityAssurance/QualityPlan.md]: Why official compliment or complaint? Maybe recorded/logged compliment or complaint? I think official is not conveying the right meaning, but not got a better phrase yet.

Comment (Chris-Hicks) [inline QualityAssurance/QualityPlan.md]: No preference removed the word official

Comment (steve-wells) [inline QualityAssurance/QualityPlan.md]: Users? who are they?  Anyone who cares to register on github , operators as well as  managers?

Comment (Chris-Hicks) [inline QualityAssurance/QualityPlan.md]: Github - No the organisation will shortly become private.  

Users are "Personnel outside of the Manufacturing Applications team".  So yes it includes operators.  They will need to write to me specifically and we will determine legitimacy and any action required

File: QualityAssurance/QualityPlan.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+- Development roadmaps and priorities will be determined by Manufacturing Applications.
+- Users have the opportunity to influence roadmaps through issues, requests and business priorities.
+
+## Exclusions
+
+The following items are excluded from this plan:
+
+- Procurement of equipment – Manufacturing Applications will identify an appropriate specification ensuring relevant performance and future coverage however it will not undertake the procurement process itself.
+- Systems introduced by third parties – Unless specifically agreed with the Manufacturing Applications Manager.
+- Hardware support - Although on occasion the Manufacturing Applications team will be involved in hardware support it is not their covered as their role or responsibility.
```
Comment (Mike-Farnhill) [inline QualityAssurance/QualityPlan.md]: "it is not their covered as their role or responsibility" should be "it is not covered in their role or responsibility"?

**Comment by Chris-Hicks** [comment]
2 review completed.  Approved by myself for release

**Pull request merged**